dispatch/api/urls.py

```diff
@@ -20,6 +20,7 @@
 router.register(r'dashboard', views.DashboardViewSet, base_name='api-dashboard')
 router.register(r'integrations', views.IntegrationViewSet, base_name='api-integrations')
 router.register(r'files',views.FileViewSet, base_name='api-files')
+router.register(r'images',views.ImageViewSet, base_name='api-images')
```

Comment:
I believe there is already a router register for this at line 17

`router.register(r'tags', views.TagViewSet, base_name='api-tags')
router.register(r'topics', views.TopicViewSet, base_name='api-topics')
router.register(r'images', views.ImageViewSet, base_name='api-images')
router.register(r'galleries', views.ImageGalleryViewSet, base_name='api-galleries')`